src/platform/datapath_raw.c

```diff
@@ -152,12 +154,15 @@ CxPlatDataPathInitialize(
         goto Error;
     }
 
+    SocketPoolInitialized = TRUE;
```

Comment:
I'm sorry for being anal here, but we've called a function named "SockPoolInitialize", not "SocketPoolInitialize".